modules/intentIqAnalyticsAdapter.js

```diff
@@ -127,89 +122,79 @@ const { BID_WON, BID_REQUESTED } = EVENTS;
+  defineGlobalVariableName();
   const iiqIdSystemConfig = getIntentIqConfig();
 
+  const options = config?.options || {}
+  const { manualWinReportEnabled, gamPredictReporting, reportMethod, reportingServerAddress: reportEndpoint, adUnitConfig, partner, ABTestingConfigurationSource } = options
+  iiqAnalyticsAnalyticsAdapter.initOptions.manualWinReportEnabled =
+            manualWinReportEnabled || false;
+  iiqAnalyticsAnalyticsAdapter.initOptions.reportMethod = parseReportingMethod(reportMethod);
+  iiqAnalyticsAnalyticsAdapter.initOptions.gamPredictReporting = typeof gamPredictReporting === 'boolean' ? gamPredictReporting : false;
+  iiqAnalyticsAnalyticsAdapter.initOptions.reportingServerAddress = typeof reportEndpoint === 'string' ? reportEndpoint : '';
+  iiqAnalyticsAnalyticsAdapter.initOptions.adUnitConfig = typeof adUnitConfig === 'number' ? adUnitConfig : 1;
+  iiqAnalyticsAnalyticsAdapter.initOptions.configSource = ABTestingConfigurationSource;
+  iiqAnalyticsAnalyticsAdapter.initOptions.currentGroup = defineABTestingGroup(options)
+
   if (iiqIdSystemConfig) {
-    const { manualWinReportEnabled, gamPredictReporting, reportMethod, reportingServerAddress: reportEndpoint, adUnitConfig } = config?.options || {}
     iiqAnalyticsAnalyticsAdapter.initOptions.lsValueInitialized = true;
     iiqAnalyticsAnalyticsAdapter.initOptions.partner =
             iiqIdSystemConfig.params?.partner && !isNaN(iiqIdSystemConfig.params.partner) ? iiqIdSystemConfig.params.partner : -1;
-
+    iiqAnalyticsAnalyticsAdapter.initOptions.abPercentage = iiqIdSystemConfig.params?.abPercentage;
     iiqAnalyticsAnalyticsAdapter.initOptions.browserBlackList =
             typeof iiqIdSystemConfig.params?.browserBlackList === 'string'
               ? iiqIdSystemConfig.params.browserBlackList.toLowerCase()
               : '';
-    iiqAnalyticsAnalyticsAdapter.initOptions.manualWinReportEnabled =
-            manualWinReportEnabled || false;
     iiqAnalyticsAnalyticsAdapter.initOptions.domainName = iiqIdSystemConfig.params?.domainName || '';
     iiqAnalyticsAnalyticsAdapter.initOptions.siloEnabled =
             typeof iiqIdSystemConfig.params?.siloEnabled === 'boolean' ? iiqIdSystemConfig.params.siloEnabled : false;
-    iiqAnalyticsAnalyticsAdapter.initOptions.reportMethod = parseReportingMethod(reportMethod);
     iiqAnalyticsAnalyticsAdapter.initOptions.additionalParams = iiqIdSystemConfig.params?.additionalParams || null;
-    iiqAnalyticsAnalyticsAdapter.initOptions.gamPredictReporting = typeof gamPredictReporting === 'boolean' ? gamPredictReporting : false;
-    iiqAnalyticsAnalyticsAdapter.initOptions.reportingServerAddress = typeof reportEndpoint === 'string' ? reportEndpoint : '';
-    iiqAnalyticsAnalyticsAdapter.initOptions.adUnitConfig = typeof adUnitConfig === 'number' ? adUnitConfig : 1;
   } else {
     logError('IIQ ANALYTICS -> there is no initialized intentIqIdSystem module')
```

Comment:
Could this be updated to a `logWarn` as it now possible to run in the configuration where the ID module is not initialized